slam-libraries/viam-orb-slam3/orbslam_server_v1_test.cc

```diff
@@ -104,13 +104,20 @@ BOOST_AUTO_TEST_CASE(
     BOOST_TEST(slamService.slam_mode == "rgbd");
 }
 
+BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_valid_config_no_data_rate_msec) {
+    const vector<string> args{"-data_dir=/path/to", "-config_param={mode=rgbd}",
+                              "-port=20000",        "-sensors=color",
+                              "-data_rate_ms=",     "-map_rate_sec=60"};
+    const string message = "No camera data rate specified";
```

Comment:
[nit] I think we should definitely mention the actual missing variable. In this case, I don't know what the "camera data rate" is referring to. Saying that `data_rate_ms` has to be defined gives clear instructions and takes out the guessing.